kalite/distributed/features/steps/login.py

```diff
@@ -33,7 +33,14 @@ def step_impl(context):
 
 @then("there should be no facility drop down")
 def step_impl(context):
-    assert not find_id_with_wait(context, "id_facility-container")
+    try:
+        # TODO(benjaoming): This is a crazy test... waiting for a timeout in
```

Comment:
One might wish to avoid the case where the element doesn't appear in 1 sec, but appears after > 1s. Obviously we can't wait forever, but 1 sec is probably reasonable until it actually causes an error. Also, `find_id_with_wait` shouldn't raise a TimeoutException, but instead just return None if no element is found within the specified time.